topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1859,6 +1859,97 @@ Now that your production-ready Galaxy is running, try registering a new user and
 
 In order to be the administrator user, you will need to register an account with the same email address you used in the group variables under the `admin_users` setting.
 
+## Job Configuration (Optional)
```

Comment:
```suggestion
## Job Configuration
```

I kinda think it should be mandatory, it's pretty quick to do this step, and helps get them on the right path for one of the first questions admins have "how can I connect it to my cluster"